auth.go

```diff
@@ -794,6 +873,18 @@ func (x *Central) UpdateIdentity(user *AuthUser) error {
 	return nil
 }
 
+// Update a user in the AuthUserStore.
+func (x *Central) UnlockAccount(user *AuthUser) error {
```

Comment:
Since the function only uses the userID, rather make the parameter int64, instead of *AuthUser